windev.ps1

```diff
@@ -20,7 +20,9 @@ function Get-LatestRelease {
         return $latestRelease.tag_name
     } catch {
         Write-Host "Error fetching release data: $_" -ForegroundColor Red
-        return $null
+        Write-Host "Redirecting to latest Full Release"
+        $latestRelease = "releases/latest/download"
```

Comment:
Thanks @Marterich, that is a cleaner way to do it than I did. 
> [!NOTE]  
> I made this PR in case of an edit to the Releases. When cleaning up some pre-releases got removed. During that time the windev file catched the issue and just gave an error. It worked, but was not the best way of doing it.

I realized that the:

```
Write-Host 'Unable to determine latest pre-release version.' -ForegroundColor Red
Write-Host "Using latest Full Release"
```

part won't be visible due to WinUtil directly launching after that and performing a clear-host.

The Windev script is not a part of winutil meaning it does not get included in the transcript. Is there a way of making that clear to the user without edits to the winutil itself? If not I think we could scrap that part to keep the script as clean as possible

Reply:
I don't mind removing the messages. If you do so, please add a short comment so that it is clear what the code does even without programming knowledge